docs/docs/reference-react-dom-server.md

```diff
@@ -15,22 +15,6 @@ import ReactDOMServer from 'react-dom/server';
 var ReactDOMServer = require('react-dom/server');
 ```
 
-We also provide a separate entry-point for use in the browser:
-
-```js
-// ES modules
-import ReactDOMServer from 'react-dom/server.browser';
```

Comment:
I removed this since it's not meant to be used directly. You'll get the right thing on the client even if you try to use `react-dom/server`.

Reply:
Oh cool, my bad!